README.md

```diff
@@ -34,3 +46,20 @@ examples of them in the [.example.env](./.example.env) and [.example.development
 ```bash
 pnpm dev
 ```
+
+## Features
+
+- **Containerized Application**: Easy setup and deployment using Docker.
+- **GitHub Authentication**: Securely log in using your GitHub account.
+- **Dynamic Portfolio Generation**: Automatically create a portfolio page with your GitHub data.
+- **Modern UI**: A sleek and responsive user interface built with SvelteKit.
+- **Real-time Data**: Fetches the latest data from your GitHub profile.
+
+## Tech Stack
+
+- **Frontend**: SvelteKit
+- **Backend**: Firebase (Authentication and Firestore)
```

Comment:
it'd be more appropriate to write
- **Frontend**: Svelte
- **Backend**: Firebase (Authentication and Firestore), SvelteKit